src/nrnmpi/mpispike.cpp

```diff
@@ -4,8 +4,10 @@
 #include <stddef.h>
 #include <assert.h>
 
+#include "../utils/profile/profiler_interface.h"
```

Comment:
Just curious - you used relative include path to avoid accidental include from some other location (e.g. coreneuron) ?

Reply:
If `src` is in the include paths, then I can also use the relative path without `..` I actually didn't check that. If we don't already pass `-Isrc` by default, then I won't change this now because I don't know what it'll do to our builds.

Reply:
> If src is in the include paths
It is added in included path

Reply:
Just verified, fixed and rebuilding - pushing in a minute